apps/meteor/app/api/server/lib/getServerInfo.spec.ts

```diff
@@ -22,7 +22,8 @@ const { getServerInfo } = proxyquire.noCallThru().load('./getServerInfo', {
 		settings: new Map(),
 	},
 });
-describe('#getServerInfo()', () => {
+// #ToDo: Fix those tests in a separate PR
```

Comment:
Is this comment still valid? should we fix it and merge into this branch?

Reply:
I just added it; those tests are currently not running, but I added new paths to the mocha config and it picked this one up, which is failing.